unime/src/lib/components/molecules/navigation/Tabs.svelte

```diff
@@ -1,36 +1,25 @@
 <script lang="ts">
   import { cubicInOut } from 'svelte/easing';
+  import type { Writable } from 'svelte/store';
   import { crossfade } from 'svelte/transition';
   import { twMerge } from 'tailwind-merge';
 
   import { createTabs, melt } from '@melt-ui/svelte';
 
+  // Bring your own store to make active tab availalbe outside.
+  export let value: Writable<string>;
   export let triggers: string[];
 
   const {
     elements: { root, list, content, trigger },
-    states: { value },
-  } = createTabs({ defaultValue: triggers[0] });
+  } = createTabs({ value });
 
   const [send, receive] = crossfade({
     duration: 250,
     easing: cubicInOut,
   });
 </script>
 
-<!--
```

Comment:
Shouldn't we document components?

Reply:
I added the comment back. I don't find it super useful because it does not show the types of the properties when hovering. I think it's easier to look at the prop names and their types to understand what a component does.

I think I read somewhere that Svelte 5 will improve the comment situation by leveraging JSDoc comments. Need to revisit then.